src/Tooltip/Tooltip.js

```diff
@@ -154,6 +163,10 @@ class Tooltip extends Util.mixin(BindMixin) {
     }
   }
 
+  triggerTooltip() {
```

Comment:
hmm, that's good. we should definitely refactor our `Form`s in future to do that rather than the whole `getTriggerSubmit` dance